ogr/abstract.py

```diff
@@ -726,6 +726,12 @@ def project_create(
         """
         raise NotImplementedError()
 
+    @property
+    def hostname(self) -> str:
+        service_repo_url = parse_git_repo(self.instance_url)
```

Comment:
I'd say it would make sense to make the result of this function call to be a property of this class so it doesn't need to be called every time someone calls hostrname, what do you think?

Reply:
yeah..thanks for the suggestion